requirements.txt

```diff
@@ -37,12 +37,14 @@ docutils==0.14
 easy-thumbnails==2.5
 elasticsearch==2.4.1
 elasticsearch-dsl==2.2.0
+gunicorn==19.7.1
 html5lib==1.0.1
 idna==2.6
 iso3166==0.8
 iso4217==1.5.20180101
 jmespath==0.9.3
 Markdown==2.6.11
+newrelic
```

Comment:
I thought that they suggested pinning this? Or is that only on upgrade? https://docs.newrelic.com/docs/agents/python-agent/hosting-services/python-agent-heroku#updating-the-python-agent

Reply:
You're right, I missed this:

> Heroku caches packages and does not detect when a newer version of New Relic's Python agent is available.

Wow. I'll pin it.